pynestml/codegeneration/resources_nest/point_neuron/common/SynapseHeader.h.jinja2

```diff
@@ -1091,6 +1094,15 @@ void {{synapseName}}< targetidentifierT >::recompute_internal_variables()
 {%- endfilter %}
 }
 
+{%- if norm_rng %}
+template < typename targetidentifierT >
+int {{synapseName}}< targetidentifierT >::get_thread()
+{
+    const thread tid = kernel().vp_manager.get_thread_id();
+    return tid;
+}
+{%- endif %}
+
```

Comment:
Can't we just use `kernel().vp_manager.get_thread_id()` directly wherever this is needed? I'm not sure I like this additional indirection... Optimally, I think `get_thread_id()` should just be available as an API function in the `nest` namespace defined in `nestkernel/nest.h` in NEST Simulator proper.